ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/arp.j2

```diff
@@ -8,3 +8,21 @@
 !
 arp aging timeout default {{ arp.aging.timeout_default }}
 {% endif %}
+{% if arp.static_entries is arista.avd.defined %}
+{# HACK - remove when dropping support for ansible 2.12.6 and its jinja2 hack #}
+{%     for entry in arp.static_entries %}
+{%         if entry.vrf is not arista.avd.defined %}
+{%             do entry.update({"vrf": "default"}) %}
+{%         endif %}
+{%     endfor %}
+{%     for vrf, entries in arp.static_entries | groupby("vrf") | arista.avd.natural_sort %}
+{# /HACK {%     for vrf, entries in arp.static_entries | arista.avd.default([]) | groupby("vrf", default="default") | arista.avd.natural_sort %} #}
```

Comment:
```suggestion
{# TODO {%     for vrf, entries in arp.static_entries | groupby("vrf", default="default") | arista.avd.natural_sort %} #}
```

Reply:
this wont work if arista.static_entries is not defined - the idea is to remove the surrounding `if` as well as it becomes useles

Reply:
OK, but then the code will change a lot, so why not just remove this line and let us implement in the best way once we get there.

Reply:
the idea was to keep track of what the correct solution should be to make the change later easier